assets/js/translator.js

```diff
@@ -741,6 +764,125 @@ function translateDoc() {
     }
 }
 
+function translateLink(href) { // eslint-disable-line no-unused-vars
```

Comment:
Is there a reason we can't just inline the contents of this function where it's used and skip the disable eslint hack?